Adafruit_INA219.h

```diff
@@ -17,202 +17,120 @@
-    INA219_CONFIG_SADCRES_12BIT_16S_8510US = (0x0060),  // 16 x 12-bit shunt samples averaged together
-    INA219_CONFIG_SADCRES_12BIT_32S_17MS   = (0x0068),  // 32 x 12-bit shunt samples averaged together
-    INA219_CONFIG_SADCRES_12BIT_64S_34MS   = (0x0070),  // 64 x 12-bit shunt samples averaged together
-    INA219_CONFIG_SADCRES_12BIT_128S_69MS =  (0x0078),  // 128 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_9BIT_1S_84US = (0x0000),   // 1 x 9-bit shunt sample
+  INA219_CONFIG_SADCRES_10BIT_1S_148US = (0x0008), // 1 x 10-bit shunt sample
+  INA219_CONFIG_SADCRES_11BIT_1S_276US = (0x0010), // 1 x 11-bit shunt sample
+  INA219_CONFIG_SADCRES_12BIT_1S_532US = (0x0018), // 1 x 12-bit shunt sample
+  INA219_CONFIG_SADCRES_12BIT_2S_1060US =
+      (0x0048), // 2 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_4S_2130US =
+      (0x0050), // 4 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_8S_4260US =
+      (0x0058), // 8 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_16S_8510US =
+      (0x0060), // 16 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_32S_17MS =
+      (0x0068), // 32 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_64S_34MS =
+      (0x0070), // 64 x 12-bit shunt samples averaged together
+  INA219_CONFIG_SADCRES_12BIT_128S_69MS =
+      (0x0078), // 128 x 12-bit shunt samples averaged together
 };
 
-/**************************************************************************/
-/*!
-    @brief  mask for operating mode bits
-*/
-/**************************************************************************/
-    #define INA219_CONFIG_MODE_MASK                (0x0007)  // Operating Mode Mask
+/** mask for operating mode bits **/
+#define INA219_CONFIG_MODE_MASK (0x0007) // Operating Mode Mask
 
-/**************************************************************************/
-/*!
-    @brief  values for operating mode
-*/
-/**************************************************************************/
- enum {
-    INA219_CONFIG_MODE_POWERDOWN          =  (0x0000),
-    INA219_CONFIG_MODE_SVOLT_TRIGGERED    =  (0x0001),
-    INA219_CONFIG_MODE_BVOLT_TRIGGERED    =  (0x0002),
-    INA219_CONFIG_MODE_SANDBVOLT_TRIGGERED = (0x0003),
-    INA219_CONFIG_MODE_ADCOFF             =  (0x0004),
-    INA219_CONFIG_MODE_SVOLT_CONTINUOUS  =  (0x0005),
-    INA219_CONFIG_MODE_BVOLT_CONTINUOUS   =  (0x0006),
-    INA219_CONFIG_MODE_SANDBVOLT_CONTINUOUS = (0x0007),
+/** values for operating mode **/
+enum {
+  INA219_CONFIG_MODE_POWERDOWN = (0x0000),
+  INA219_CONFIG_MODE_SVOLT_TRIGGERED = (0x0001),
+  INA219_CONFIG_MODE_BVOLT_TRIGGERED = (0x0002),
+  INA219_CONFIG_MODE_SANDBVOLT_TRIGGERED = (0x0003),
+  INA219_CONFIG_MODE_ADCOFF = (0x0004),
+  INA219_CONFIG_MODE_SVOLT_CONTINUOUS = (0x0005),
+  INA219_CONFIG_MODE_BVOLT_CONTINUOUS = (0x0006),
+  INA219_CONFIG_MODE_SANDBVOLT_CONTINUOUS = (0x0007),
```

Comment:
Since these start at zero and are contiguous, I think you can skip the assignment.

Reply:
yeah, I changed It :)

Reply:
tested It on UNO to measure battery and works fine after changes